tests/utils/test_file_lookups.py

```diff
@@ -50,58 +49,51 @@ def test_project_id_exception(mock_gitlab):
 @mock.patch('foremast.utils.lookups.gitlab')
 def test_project_id_success(mock_gitlab):
     """Check resolving GitLab Project ID is successful."""
-    project_id = 30
-
-    mock_gitlab.Gitlab.return_value.getproject.return_value = {'id': project_id}
-
+    project_id = object
+    mock_gitlab.Gitlab.return_value.projects.get.return_value = object
     seeker = FileLookup()
-
-    assert seeker.project_id == project_id
+    assert seeker.project == project_id
```

Comment:
This is testing equivalency. You should test that they are identical with `is`. The return value of `...projects.get()` should be identical to `self.project`.

Reply:
Fixed.